docs/SettingUp.md

```diff
@@ -0,0 +1,64 @@
+# Duke project template
```

Comment:
Is this a copy of the main README?

Reply:
Yes the main README is outside the directory of the tp folder hence it can't be accessed via gh pages, i've tried